optaplanner-operator/optaplanner-operator-integration-test-parent/optaplanner-operator-integration-test/src/test/resources/school-timetabling-solver.yml

```diff
@@ -0,0 +1,25 @@
+# Example solver resource; override as you see fit.
+apiVersion: ${solver.api.version}
+kind: Solver
+metadata:
+  name: school-timetabling
+spec:
+  amqBroker:
+    host: ex-aao-amqp-0-svc.demo.svc.cluster.local
+    port: 5672
+    managementHost: ex-aao-hdls-svc.demo.svc.cluster.local
+    usernameSecretRef:
+      key: AMQ_USER
+      name: ex-aao-credentials-secret
+    passwordSecretRef:
+      key: AMQ_PASSWORD
+      name: ex-aao-credentials-secret
+  template:
+    spec:
+      containers:
+        - name: school-timetabling
+          image: quay.io/optaplanner/school-timetabling:${project.version}
```

Comment:
For the "full scenarios": where do we take the image from? I don't particularly like the idea of exposing a testing image in `quay.io`.

Reply:
Yes please, the current image name is misleading.